build/__tools__/run-perf-tests.sh

```diff
@@ -34,7 +34,7 @@ F_ITERATIONS=1000
 F_LEDGERURL=--no-ledger
 F_LOGLEVEL=info
 F_SERVICES=1
-F_SERVICEURL=http://localhost:7101
+F_SERVICEURL=http://intsim-t4.jf.intel.com:7101
 F_USAGE='[-c|--count client] [-i|--iterations count] [--ledger url]'
 
```

Comment:
could we please remove intsim-t4.jf.intel.com

Reply:
uggh... yes. this should have been PDO_HOSTNAME. will fix.

Reply:
done